StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Embedded/EmbeddedPaymentElement.swift

```diff
@@ -209,13 +209,12 @@ public final class EmbeddedPaymentElement {
         analyticsHelper.log(event: .mcConfirmEmbedded)
         guard let presentingViewController else {
             let errorMessage = "Presenting view controller is nil. Please set EmbeddedPaymentElement.presentingViewController."
-            stpAssertionFailure(errorMessage)
+            assertionFailure(errorMessage)
```

Comment:
Why not stpAssertionFailure? it calls assertionFailure under the hood

Reply:
https://github.com/stripe/stripe-ios/blob/8a549e7c/StripeCore/StripeCore/Source/Helpers/STPAssert.swift#L20-L21

```
/// A wrapper that only calls `assertionFailure` when the `ENABLE_STPASSERTIONFAILURE` compiler flag is set.
/// Use this for assertions that should not trigger in merchant apps.
```